spec/api/sul_bib/authorship_api_spec.rb

```diff
@@ -263,9 +217,7 @@
     it_behaves_like 'it is successful'
 
     it 'does not create a new contribution' do
-      expect do
-        http_request # defined in context, uses request_data
-      end.not_to change(Contribution, :count)
+      expect { http_request }.not_to change(Contribution, :count)
     end
```

Comment:
The explanatory comment was there to dispel mysteries, esp. for junior devs or anyone not familiar with shared examples or let precedence.  It can also help with finding any lines related to `request_data`.  Could we find a way to accomplish this (if it's not done elsewhere)?

Reply:
Sorry, what are you referring to?  You're commenting on an unchanged line.   The comment above (L186/R163) was changed only inasmuch as the keys were corrected to be strings (not symbols).  Are you describing something else?